### PR TITLE
cli: Update Arm Compiler 6 paths

### PR DESCRIPTION
**Pull request by arekzaluski**
# Changes:

- Update path to Arm Compiler 6

### CODE REVIEW

**Comment by iriark01** [comment]
Do not merge until 0.5 goes out

**Review by GuyWi — APPROVED** [review]

**Review by AnotherButler — APPROVED** [review]

File: docs/tools/CLI/cli-setup/cli-reqs.md [hunk 1]
```diff
@@ -233,14 +233,19 @@ Mbed CLI supports a setting for each toolchain path:
...
-    - Mac: `/Library/Application Support/Mbed Studio/mbed-studio-tools`
+    - Windows: `C:\Users\<user>\AppData\Local\Mbed Studio\.mbed-studio-tools\ac6\bin`
+    - Mac: `/Library/Application Support/Mbed Studio/mbed-studio-tools/ac6/bin`
+
+1. Set the environment variable `ARMLMD_LICENSE_FILE`: 
 
-1. Set the environment variable `ARMLMD_LICENSE_FILE` to the path of Mbed Studio's Arm Compiler 6 license. For Mbed Studio 0.5 and later, the license is located at `ac6-license.dat`. This file is relative to the Mbed Studio application data defined above.
+    For Mbed Studio 0.5 and later, the Arm Compiler 6 license file, `ac6-license.dat`, is located in the following path:
+
+    - Windows: `C:\Users\<user>\AppData\Local\Mbed Studio\.mbed-studio-tools`
```
Comment (arekzaluski) [inline docs/tools/CLI/cli-setup/cli-reqs.md]: @GuyWi There is an extra `.` in the path. It should be replaced to `C:\Users\<user>\AppData\Local\Mbed Studio\mbed-studio-tools`

Comment (arekzaluski) [reply docs/tools/CLI/cli-setup/cli-reqs.md]: Same above

Comment (AnotherButler) [reply docs/tools/CLI/cli-setup/cli-reqs.md]: Fixed

**Comment by GuyWi** [comment]
Thanks for handling this, @AnotherButler ! :)

**Pull request merged**